repo/fsrepo/fsrepo.go

```diff
@@ -562,7 +562,9 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 
 	var du uint64
 	err = filepath.Walk(pth, func(p string, f os.FileInfo, err error) error {
-		du += uint64(f.Size())
+		if f != nil {
```

Comment:
but the error inside this function is different than the error returned from `filepath.Walk`. The error passed into the function here is one that comes from calling stat on the path we're also passed. The one from `filepath.Walk` is whatever we return from our function